doc/radosgw/config-ref.rst

```diff
@@ -40,7 +40,7 @@ instances or all radosgw-admin options can be put into the ``[global]`` or the
                         participate in a `multi-site <../multisite>`_
                         configuration.
 :Type: String
-:Default: ``s3, swift, swift_auth, admin`` All APIs.
+:Default: ``s3, swift, swift_auth, admin, notifications`` All APIs.
```

Comment:
from https://github.com/ceph/ceph/blob/master/src/common/options.cc#L5723-L5725:
```c++
    Option("rgw_enable_apis", Option::TYPE_STR, Option::LEVEL_ADVANCED)
    .set_default("s3, s3website, swift, swift_auth, admin, sts, iam, pubsub")
```

Reply:
its fine to rename `pubsub` to `notifications` there too. but this doc should match exactly what's in options.cc

Reply:
can you please update this Default to match what's in options.cc?

Reply:
@cbodley yep updated and renamed pubsub to notifications here.

Reply:
i see that you updated options.cc - but i'd like to see this `:Default:` in config-ref.rst updated to match

Reply:
@cbodley done, matched and updated.